manifests/cluster-infra/3b_prometheus-meta.yaml

```
@@ -130,7 +132,7 @@ spec:
         args:
         - "--config.file=/etc/prometheus/config/prometheus.yaml"
         - "--storage.tsdb.path=/data"
-        - "--storage.tsdb.retention=90d"
+        - "--storage.tsdb.retention.size=800GB"  #set to 80% of the storage available
```

put the same comment in the yaml file where we set the current disk size.
something like:
1000GB # If you change this make sure to update the prometheus meta disk retention settings.

also lets make it at 50% as when we add logging with loki it will use quite a bit of space as well.
```suggestion
        - "--storage.tsdb.retention.size=500GB"  # Eet to 50% of the total storage available.
```

---

No I meant leave this one here as an addition to the one in the other yaml file.